CHANGELOG.md

```diff
@@ -3,61 +3,52 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## `Unreleased`
+## v0.2.0 - [2024-05-14]
 
 ### `Added`
 
-- Updated documentation for params.
+- Updated documentation for params. See [PR 66](https://github.com/phac-nml/mikrokondo/pull/66)
```

Comment:
I switched the format of the changelog so it references the PRs implementing each change